Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lin-bus-driver-serial"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Since we bump a public dependency with a semver major bump, we should also bump the version to 0.2 here.

Reply:
yup, makes sense, thanks